DesignDoc/RMA.md

```diff
@@ -1,3 +1,205 @@
 # Request Merchandise Authorization (RMA) notification structure design
 
-## TBA
+## Intent
+
+Structure for RMA process.
+
+## Definitions
+
+- Consumer : Customers of device vendors (Eg: Google, Microsoft, X).
+- Publisher: Device vendor (Ciena, Adva, Infinera, X).
+
+## Features 
+
+RMA case is raised by the consumer to request the replacement of the defective parts. It consists of the following subflows:
+1. Consumer starts to request the RMA from the Publisher, and wait for the review/approval from the publisher
```

Comment:
Let's split this into following points:
1. Consumer starts to request the RMA from Publisher
1. Consumer waits for the review from the Publisher
1. Consumer responds any pending review request from the Publisher till RMA is approved
1. Publisher approves the RMA request
1. Consumer waits for RMA shipment update from the Publisher

Reply:
fixed as the suggestions.